README.md

```diff
@@ -79,140 +62,125 @@ $ open QuickStart.xcworkspace
+               }
+                   responseHandler:^(NSURLResponse *response, NSData *responseData, NSError *error) {
+                       // Process the response
+                       if (responseData) {
+                           NSDictionary *dataReturned = [NSJSONSerialization JSONObjectWithData:responseData options:0 error:nil];
+                           NSLog(@"Graph Response was: %@", dataReturned);
+                           
+                           // We can grab the top most JSON node to get our graph data.
+                           NSArray *graphDataArray = [dataReturned objectForKey:@"value"];
+                           
+                           // Don't be thrown off by the key name being "value". It really is the name of the
+                           // first node. :-)
+                           
+                           //each object is a key value pair
+                           NSDictionary *keyValuePairs;
+                           NSMutableArray* Users = [[NSMutableArray alloc]init];
+                           
+                           for(int i =0; i < graphDataArray.count; i++)
+                           {
+                               keyValuePairs = [graphDataArray objectAtIndex:i];
+                               
+                               User *s = [[User alloc]init];
+                               s.upn = [keyValuePairs valueForKey:@"userPrincipalName"];
+                               s.name =[keyValuePairs valueForKey:@"displayName"];
+                               s.mail =[keyValuePairs valueForKey:@"mail"];
+                               s.businessPhones =[keyValuePairs valueForKey:@"businessPhones"];
+                               s.mobilePhones =[keyValuePairs valueForKey:@"mobilePhone"];
+                               
+                               
+                               [Users addObject:s];
+                           }
+                           
+                           completionBlock(Users, nil);
+                       }
+                       else
+                       {
+                           completionBlock(nil, error);
+                       }
+                       
+                   }];
 }
-
 ```
-- When your app requests a token by calling `getToken(...)`, ADAL will attempt to return a token without asking the user for credentials.  If ADAL determines that the user needs to sign in to get a token, it will display a login dialog, collect the user's credentials, and return a token upon successful authentication.  If ADAL is unable to return a token for any reason, it will throw an `AdalException`.
-- Notice that the `AuthenticationResult` object contains a `tokenCacheStoreItem` object that can be used to collect information your app may need.  In the QuickStart, `tokenCacheStoreItem` is used to determine if authenitcation has already occurred.
 
 
 ## Step 5: Build and Run the application
```

Comment:
This is a different formatting than the first 4 paragraph (which were using # 5.)